src/js/state/Boards/index.ts

```diff
@@ -0,0 +1,30 @@
+import {createEntityAdapter, createSlice, PayloadAction} from "@reduxjs/toolkit"
+import {State} from "../types"
+
+export type Board = {id: string; title: string; tiles: string[]}
+export type BoardsState = ReturnType<typeof slice.reducer>
+
+const adapter = createEntityAdapter<Board>()
```

Comment:
It was useful for me to first understand [Create Action](https://redux-toolkit.js.org/api/createAction), [Create Reducer](https://redux-toolkit.js.org/api/createReducer), and [Create Slice](https://redux-toolkit.js.org/api/createSlice) as they each build on one another.

Docs for create entity adapter: https://redux-toolkit.js.org/api/createEntityAdapter